### PR TITLE
Project R Publikator needs to die, the other part

### PR DESCRIPTION
**Pull request by annatraussnig**

### CODE REVIEW

File: packages/styleguide/src/components/TeaserFeed/index.js [hunk 1]
```diff
@@ -17,31 +17,34 @@ const dateFormat = timeFormat('%d.%m.%Y')
...
 const creditSchema = {
-  rules: [link, br]
+  rules: [link, br],
 }
 
-const DefaultLink = ({ children, path }) => children
+const DefaultLink = ({ children, href }) => children
+
+export const getTeaserHref = (path, externalUrl) =>
+  externalUrl ? externalUrl.replace(/\/$/, '').concat('/').concat(path) : path
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply packages/styleguide/src/components/TeaserFeed/index.js:47]: aha, I didn't know we had such a convention.

[the remark this answers is not in the document]
Comment (annatraussnig) [reply packages/styleguide/src/components/TeaserFeed/index.js]: done

File: apps/www/components/Feed/DocumentListContainer.js [hunk 2]
```diff
@@ -97,6 +99,7 @@ const DocumentListContainer = ({
       error={error}
       render={() => {
         const connection = getConnection(data)
+        console.log({ connection })
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply apps/www/components/Feed/DocumentListContainer.js]: I'm innocent 🙈

**Comment by annatraussnig** [comment]
I did the 404, but maybe a redirect would make more sense? Otherwise we also need to edit the repo search & link function in Publikator to accommodate for externalBaseUrl. We don't need this with a redirect, but maybe we don't want to redirect to outside the republik domain?

**Comment by tpreusse** [comment]
Mmmmh hard call. Tending to 404 and adapting link resolve in the backend—Publikator does repo links as is and backend resolves them to a full url with `externalBaseUrl` baked in. @patrickvenetz is that feasible to resolve in the backend? I guess might be a bit more backend work than anticipated—maybe we should even extend `document` and `redirect` query to also send a base url up—not returning documents for wrong bases and support redirects going to other domains?

**Pull request merged**